### ISSUE TITLE
Upgrade guide shows example bower.json file, but the version of webcomponentsjs isn't available

### ISSUE THREAD

**Issue by akc42** [Polymer/docs#2147]
In order to make my bower.json file work with Polymer 2 I had to use 

`"webcomponentsjs": "webcomponents/webcomponentsjs#^v1.0.0-rc.11"`

in my bower.json.  The equivalent line in the Polymer 2.0 upgrade guide doesn't work

**Comment by arthurevans** [comment]
Hm. "webcomponents/webcomponentsjs#^v1.0.0" works fine for me locally. Have you tried removing the `bower_components` folder to make sure you don't have any old dependencies, and removing any existing `resolutions` in `bower.json` that might point to older components?

**Comment by akc42** [comment]
I think I got to the bottom of the problem.  I didn't have `"interactive": false` in my `.bowerrc` file and as a result bower was hanging (waiting for input), except I just didn't know it, since by then the log lines and wizzed past.  This just happened to be where it appeared to hang.

The reason it was asking for input is that I had had to manually add each individual `iron-*` and `paper-*` entry rather than use the overall imports I used with polymer1 and I had made some typos.

Sorry for the noise.
